orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/MonitoredDeployBaseTask.java

```diff
@@ -188,6 +189,22 @@ private TaskResult handleError(
       }
     }
 
+    // Don't let failures in NotifyDeployComplete cause rollback
+    if (stage.getType().equalsIgnoreCase(NotifyDeployCompletedStage.PIPELINE_CONFIG_TYPE)) {
```

Comment:
Can you elaborate on how this part is preventing rollback in case one of the 3 stages fails?

Reply:
this is part of it, this will cause, e.g. http errors to monitor service to bubble as `FAILED_CONTINUE` instead of `TERMINAL` then the changes https://github.com/spinnaker/orca/pull/3329/files#diff-cc03fa80e1a96b9c0443a8fe1ef22a6fR349-R351 will instruct orca to continue with other stages